ledger/src/blockstore_db.rs

```diff
@@ -955,10 +989,40 @@ pub struct WriteBatch<'a> {
     map: HashMap<&'static str, &'a ColumnFamily>,
 }
 
+pub enum ShredStorageType {
```

Comment:
nit: rename to `ShredCompactionType`

Reply:
Commented in another question.  This is to give us the flexibility to also switch to other storage types, such as Steve's custom storage for instance.